guide/preferences.md

```diff
@@ -40,6 +40,12 @@ depending on the GPU OpenGL abilities.
 the Camera device images. The methods are sorted from the most efficient one to
 the least efficient one.
 
+- The `Main 3D View Anti-Aliasing` option allows you to enable Multisample Anti-Aliasing
```

Comment:
Is it required to introduce upper case here (I mean for the `Multisample Anti-Aliasing` term)?